src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -533,8 +533,8 @@ private IObjectOrCollectionInitializerExpression CreateBoundCollectionInitialize
         private IMemberReferenceExpression CreateBoundObjectInitializerMemberOperation(BoundObjectInitializerMember boundObjectInitializerMember)
         {
             Lazy<IOperation> instance = new Lazy<IOperation>(() => new InstanceReferenceExpression(
-                InstanceReferenceKind.Implicit,
-                _semanticModel,
+                isBaseReference: false,
```

Comment:
Actually, I am not sure if we are constructing this node in the right way here. Should it's type be derived from the member? That type could be a base type rather than the type being initialized (```this``` type).

---
In reply to: [134357695](https://github.com/dotnet/roslyn/pull/21625#discussion_r134357695) [](ancestors = 134357695)